mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookGhostSlots.mapping

```diff
@@ -9,6 +9,7 @@ CLASS net/minecraft/class_505 net/minecraft/client/gui/screen/recipebook/RecipeB
 		ARG 1 client
 		ARG 2 x
 		ARG 3 y
+		ARG 4 isBig
```

Comment:
Like the output slot or?

Reply:
yes, it is to draw the ghost slot background (the red overlay) bigger, but it doesn't have to be an output slot.